Packs/ServiceNow/Integrations/ServiceNowv2/ServiceNowv2.py

```diff
@@ -839,11 +841,11 @@ def update_ticket_command(client: Client, args: dict) -> Tuple[Any, Dict, Dict,
     ticket_id = str(args.get('id', ''))
     additional_fields = split_fields(str(args.get('additional_fields', '')))
     additional_fields_keys = list(additional_fields.keys())
-
     fields = get_ticket_fields(args, ticket_type=ticket_type)
     fields.update(additional_fields)
+    input_display_value = args.get('fields', 'input_display_value') == 'true'
```

Comment:
Use `str2bool`

Reply:
str2bool is in the sdk, used `argToBoolean`